torch/_decomp/__init__.py

```diff
@@ -224,6 +224,8 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten.index_copy_,
             aten.index_fill,
             aten.index_fill_,
+            aten.isposinf,
+            aten.isneginf,
             aten.l1_loss,
```

Comment:
Do we need to add `isinf` here as well?

Reply:
> `isinf` can also be written with aten functions, however I don't add
it to inductor's decompositions because `isinf` is lowered to
`tl.libdevice.isinf` in triton.